packages/remote-replay-launcher/src/upload-assets-and-trigger-test-run.ts

```diff
@@ -97,6 +97,20 @@ export const uploadAssetsAndTriggerTestRun = async ({
     throw new Error(`Directory does not exist: ${resolvedAppDirectory}`);
   }
 
+  // If there are no rewrites, check for an index.html in the root directory
+  if (!rewrites || rewrites.length === 0) {
+    const indexHtmlPath = join(resolvedAppDirectory, "index.html");
+    const indexHtmPath = join(resolvedAppDirectory, "index.htm");
```

Comment:
`index.htm` won't help if they don't have any rewrites because the default rewrite rule we use rewrites to `index.html`.